rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -0,0 +1,899 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hsearch" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis for migration from Hibernate Search 4.x to Hibernate Search 5.x.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.4.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
+        </dependencies>
+        <sourceTechnology id="hibernate-search" versionRange="[4,)" />
+        <sourceTechnology id="eap" versionRange="[6,)" />
```

Comment:
Both `<sourceTechnology>`-ies should have an upper limit on the version. I know, I know, the ranges don't really mean anything, but if they're there, they should be correct.

```xml
<sourceTechnology id="hibernate-search" versionRange="[4,5)" />
<sourceTechnology id="eap" versionRange="[6,7)" />
```